repairnator/repairnator-realtime/src/main/java/fr/inria/spirals/repairnator/realtime/RTLauncher.java

```diff
@@ -164,12 +166,33 @@ private JSAP defineArgs() throws JSAPException {
         jsap.registerParameter(opt2);
 
         opt2 = new FlaggedOption("numberofpatchedbuilds");
-        opt2.setLongFlag("numberofatchedbuilds");
```

Comment:
nice spot! would you create a separate PR for this?

Reply:
I'll do that later :) .